R/correctionsataglance-data.R

```diff
@@ -101,13 +101,14 @@ formatDateRange <- function(startD, endD){
     endSeq <- c(startSeq[-1], endD)
   }
   
-  #don't print Month Year in plot if there isn't enough room inside the rectangle
+#   #don't print Month Year in plot if there isn't enough room inside the rectangle
   dateSeq <- startSeq
   startSpan <- as.numeric(endSeq[1] - startSeq[1])
   endSpan <- as.numeric(tail(endSeq, 1) - tail(startSeq, 1))
-  if(startSpan <= 16){dateSeq[1] <- NA}
-  if(endSpan <= 14){dateSeq[length(dateSeq)] <- NA}
-  
+  startLabelText <- format(startD, "%b %Y")
+  endLabelText <-  format(endD, "%b %Y")
+  #if(isTextLong(labelText=startLabelText,dateLim=NULL,startD,endD,totalDays=numdays)) {dateSeq[1] <- NA}
```

Comment:
Yeah I couldn't decide on if this is actually used to not plot the first month so I commented it out. Do you remember why you would want to shorten the start of the approval span? maybe that is the problem I'm seeing in AQCU-664, where it's shifting the labels on single month ranges... hmm.